modules/ROOT/pages/quotas-and-limits.adoc

```diff
@@ -28,7 +28,7 @@ a|
 | 128 characters
 | Languages 
 a| 
-Full English Support +
+Full English, Spanish, and German Support +
```

Comment:
Do we need the comma after Spanish?

Reply:
Yes, we use the Oxford comma per our guidelines.